src/Test.UnitTests.Sarif.Converters/PoliCheckConverterTests.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using FluentAssertions;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.Converters
+{
+    public class PoliCheckConverterTests
+    {
+        [Fact]
+        public void PoliCheckConverter_Convert_NullInput()
+        {
+            Assert.Throws<ArgumentNullException>(() => new PoliCheckConverter().Convert(null, null, OptionallyEmittedData.None));
```

Comment:
i should pass valid output here so that the test is targeted to input null check only